Automattic/wp-calypso
# Hosting Command Palette: Add a command to send feedback

Related to https://github.com/Automattic/dotcom-forge/issues/4667

## Proposed Changes

This PR adds a command to `Send feedback` by opening an email option in the help center. 

## Testing Instructions

* Navigate to Calypso Live Link
* Press `cmd + k` on macOS or `ctrl + k` on Windows to open the command palette
* Search for `Send feedback` command:

<img width="962" alt="Screenshot 2023-12-18 at 5 30 25 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/60472d39-f668-4474-a15d-df7037efc180">

* Select the command and observe that it opens the help center with the email option:

<img width="975" alt="Screenshot 2023-12-18 at 5 30 34 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/7982d5eb-63b3-4621-95d1-6e32a2f328e2">

* Fill in the subject and the feedback
* Send email
* Navigate to Zendesk
* Look for the ticket that you sent (the easiest way is to search for the ticket subject that you typed by using the search icon in the top right corner and filtering by the most recent tickets - see screenshots below):

<img width="1291" alt="Screenshot 2024-01-10 at 10 45 37 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/d3e984bc-c1fe-4e42-85c5-e9cc0482e384">

<img width="529" alt="Screenshot 2024-01-10 at 10 49 48 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/0bdb03b9-b2db-4989-b23a-236980a3c0b9">

* Confirm that you can see the text added to the ticket meta:

<img width="573" alt="Screenshot 2024-01-10 at 10 47 23 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/1999c4be-4815-469f-b4d0-5f3389e98b85">

Please note that if you open the Help Center, go to the fields to fill in the feedback, use the `Back` button and then navigate back to fill in the feedback, this custom text won't be added to the ticket meta (because the custom parameters that we set in the flow would be reset). We discussed this flow with @sejas and since this is a bit of an edge case, we decided to leave it as is for now. 

## Pre-merge Checklist
